src/modules/navigator/mission_block.cpp

```diff
@@ -307,6 +308,19 @@ MissionBlock::is_mission_item_reached()
 				mission_acceptance_radius = _navigator->get_acceptance_radius();
 			}
 
+			/* for vtol back transition calculate acceptance radius based on time and ground speed */
+			if (_mission_item.vtol_back_transition) {
+
+			    float groundspeed = sqrtf(_navigator->get_global_position()->vel_n * _navigator->get_global_position()->vel_n +
```

Comment:
@sanderux set this option https://github.com/PX4/sitl_gazebo/blob/master/CMakeLists.txt#L146

Can be modified here - https://github.com/PX4/sitl_gazebo/blob/master/models/standard_vtol/standard_vtol.sdf#L853